README.md

```diff
@@ -22,9 +22,9 @@ Start pygmy: `pygmy up`
 ## Create a new project
 
 ``` shell
-docker run --rm -it -v $PWD:/app --user $(id -u):$(id -g) composer \
+docker run --rm -it -v $PWD:/app --user $(id -u):$(id -g) tamasd/composer \
```

Comment:
As it is a tempory solution I am fine having this under your namespace instead of Pronovix.